x-pack/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/nodes/match_prebuilt_rule/match_prebuilt_rule.ts

```diff
@@ -51,6 +52,7 @@ export const getMatchPrebuiltRuleNode =
           description: result.rule.description,
           prebuilt_rule_id: result.rule.rule_id,
           id: result.installedRuleId,
+          translation_result: SiemMigrationRuleTranslationResult.FULL,
```

Comment:
The `translation_result` goes outside `elatic_rule`, I wonder how the linter accepted this 🤔 
I'll fix it on the following backport